src/sync/types.rs

```diff
@@ -30,6 +32,8 @@ pub struct MaybeEndTryPullResponse {
     pub replay_mutations: Vec<ReplayMutation>,
     #[serde(rename = "syncHead")]
     pub sync_head: String,
+    #[serde(rename = "changedKeys")]
+    pub changed_keys: ChangedKeysMap,
```

Comment:
Maybe say here that changed_keys will only be filled if A) the caller requested it and B) on the last MaybeEndTryPull call, when there are no more to replay.